examples/custom-proxy-url/main.tf

```diff
@@ -4,5 +4,5 @@ module "lacework_k8s_datacollector" {
   source = "../../"
 
   lacework_access_token = "0123456789ABCDEF0123456789ABCDEF"
-  lacework_proxy_url   = "https://proxy.address.here:8080"
+  lacework_proxy_url    = "https://proxy.address.here:8080"
```

Comment:
Could you remove this change from the PR?

Reply:
https://github.com/lacework/terraform-kubernetes-agent/pull/72

Merge that one, then I'll rebase or something.